iot-e2e-tests/android/app/build.gradle

```diff
@@ -82,10 +82,12 @@ dependencies {
     implementation 'com.android.support:appcompat-v7:28.0.0'
     implementation 'com.android.support:multidex:1.0.3'
 
-    implementation ('com.microsoft.azure.sdk.iot:iot-e2e-common:0.26.0'){
```

Comment:
In turning the common module into a test project, this jar no longer contains the actual test code that android needs to run. Switching over to local references because there is no other way to get the "iot-e2e-common-0.26.0-tests.jar" file